devito/symbolics/printer.py

```diff
@@ -20,8 +20,7 @@ class CodePrinter(C99CodePrinter):
         'FLOAT': '(float)',
         'FLOATP': '(float*)',
         'DOUBLE': '(double)',
-        'DOUBLEP': '(double*)',
-    }
+        'DOUBLEP': '(double*)'}
```

Comment:
leftover... always check the Files Changed panel

Reply:
Sure